### ISSUE TITLE
Bales don't disappear on map

### ISSUE THREAD

**Issue by Maag000** [Mmtrx/FS22_SeeBales#4]
If I use SeeBales with the mod Selectable Bale Capacity from Modhub, the bale icons on the map don't disappear if I collect them.
Can you please fix this problem?

**Comment by Mmtrx** [comment]
Ja, will look into it. Together with additions to provide compatibility with Maize+.
I will release a first test version here before end of the month.

**Comment by Gordon-Dry** [comment]
I would like to see that test version.
It also bothers me that the bale markers are still shown on the last position on the map after they have been collected, until they are unloaded again.
